Resources/public/js/views/ez-subitemlistview.js

```diff
@@ -160,6 +173,207 @@ YUI.add('ez-subitemlistview', function (Y) {
+            input.set('value', subItem.location.get('priority'));
+            input.setAttribute('readonly', 'readonly');
+        },
+
+        /**
+         * Return the sub item with the giiven locationId
+         *
+         * @method _getSubItem
+         * @protected
+         * @param {Number} locationId
+         */
+        _getSubItem: function (locationId) {
+            return Y.Array.find(this.get('subitems'), function (locStruct) {
+                return locStruct.location.get('locationId') == locationId;
+            });
+        },
+
+        /**
+         * Hide the validate and cancel buttons displayed while editing priority
+         *
+         * @method _hidePriorityButtons
+         * @protected
+         */
+        _hidePriorityButtons: function () {
+            var selectedCell = this.get('container').one('.ez-subitem-selected-priority-cell');
+
+            selectedCell.removeClass('ez-subitem-selected-priority-cell');
+            this._set('isOnePrioritySelected', false);
+        },
+
+        /**
+         * Event handler allowing to change the priority of the subitem
+         *
+         * @method _setPriority
+         * @protected
+         * @param {Object} e event facade
+         */
+        _setPriority: function (e) {
+            var button = e.target,
+                selectedCell = this.get('container').one('.ez-subitem-selected-priority-cell'),
+                input = this._getPriorityInput('#' + button.getAttribute('data-priority-input')),
+                subItem = this._getSubItem(input.getAttribute('data-location-id'));
+
+            e.preventDefault();
+
+            this.fire('updatePriority', {location: subItem.location, priority: input.get('value')});
+            input.setAttribute('readonly', 'readonly');
+            selectedCell.removeClass('ez-subitem-selected-priority-cell');
+            this._set('isOnePrioritySelected', false);
+        },
+
+        /**
+         * Validates the current input of priority
+         *
+         * @method _validatePriority
+         * @param {Object} e event facade
+         */
+        _validatePriority: function (e) {
+            var validity = this._getInputValidity(e.target),
+                selectedCell = this._getPriorityCell(e.target.getAttribute('data-location-id'));
+
+            if ( validity.patternMismatch || validity.valueMissing ) {
+                this._displayErrorIcon(e.target, selectedCell);
+            } else if (selectedCell.hasClass('ez-subitem-error-priority-cell')) {
+                this._hideErrorIcon(e.target, selectedCell);
+            }
+        },
+
+        /**
+         * Return the currently selected priority cell
+         *
+         * @method _getPriorityCell
+         * @param {String} locationId
+         */
+        _getPriorityCell: function (locationId) {
+            return this.get('container').one('#priority-cell-' + locationId);
+        },
+
+        /**
+         * Return the current priority input
```

Comment:
*current* ? no, it returns a priority input from its id. (missing doc on the return value).